pom.xml

```diff
@@ -219,4 +219,13 @@
 
         </profile>
     </profiles>
+
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
```

Comment:
We don't need to add any dependency in the root `pom.xml`. Do you know why you needed to add here?

Reply:
Because I wanted to execute all the tests by the contextual menu of the parent > "Run all tests" and I had an error message that said that junit.rar was missing. I searched on Google and found this solution. After that I ran the tests without any problem. Isn't it a good solution ?

Reply:
The best solution to execute all tests is to run Maven command : `mvn clean package`. The root `pom.xml` must not contain any dependency. 

You should execute tests from module (`symphony-api-client-java`) only

Reply:
Should I make a ticket, assign it to me and resolve it ?

Reply:
No it's ok @Haiddi-Avencod, you should just revert this change in root `pom.xml` and run `mvn clean package` from your terminal when you want to execute all tests

Reply:
Here's the result when I run `mvn clean package`
![mvn_clean_package](https://user-images.githubusercontent.com/68892288/93430771-77c60a00-f8c3-11ea-91b3-10e03d9839b9.JPG)

Reply:
I fixed it ;)